### DISCUSSION TITLE
Should articles have a featured image? (in version 2)

### DISCUSSION THREAD

**Discussion by louiechristie** [realworld-apps/realworld#979] [Polls]
Most blogs, including Medium, have a featured image for each article (optional, sometimes known as a 'hero' image). To use conduit as a Real World blog...

**Comment by geromegrignon** [comment]
Hi @louiechristie,

Sorry, I totally missed this poll.
I want to add it, not just for the sake of having a hero image but as an opportunity to include file upload in the project. That's a common real-world feature.

I still need to evaluate it, as it should be included in the demo. As so, it requires having an object storage solution.
As I'm paying for the project servers/domain, I either need a really cheap solution or to find some funding partner.
